File: SoccerApp/Domain/Entities/Player.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Entities
{
    public enum Position
    {
        
        Goalkeeper,
        Defender,
        Midlfielder,
        Forward
    }
    public class Player
    {
        public int PlayerId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public double Sallary { get; set; }
        public Position Position { get; set; }
        public CCountry Country { get; set; }
        public int? CountryId { get; set; }
        public Team Team { get; set; }
        public int? TeamId { get; set; }
    }
}

File: SoccerSPA/SoccerSPA/src/app/models/player.ts
import { country } from "./country";
import { team } from "./team";

export interface player{
  playerId?:number;
  firstName?:string;
  lastName?:string;
  dateOfBirth?:Date;
  sallary?:number;
  position?:string;
  country?:country;
  team?:team;
  teamId?:number;

}
